backstack/build.gradle.kts

```diff
@@ -16,9 +20,12 @@ kotlin {
   iosX64()
   iosArm64()
   iosSimulatorArm64()
-  js {
+  js(IR) {
     moduleName = property("POM_ARTIFACT_ID").toString()
-    nodejs()
+    browser()
+  }
+  if (hasProperty("enableWasm")) {
+    @OptIn(ExperimentalWasmDsl::class) wasmJs { browser() }
```

Comment:
Does the `wasmJs` need the `moduleName` here?

Reply:
ah yes good catch

Reply:
[a9cfa95](https://github.com/slackhq/circuit/pull/1209/commits/a9cfa95a72688c1a9df5bc2d6f0f810bde6f889d)